src/_data/service_lines_tabber.yml

```diff
@@ -133,34 +133,34 @@ es:
         - Reducción de costes
         - Actualización tecnológica
         - Mitigación frente al end-of-life
-      details: Cloud native; arquitectura sin servidor; integración de terceros; implementación en contenedores; escalamiento elástico; seguridad; automatización;Infrastructure-as-code; Managed databases; Modern frameworks Modular architecture
+      details: Cloud native; Arquitectura sin servidor; Integración de terceros; Implementación en contenedores; Escalamiento elástico; Seguridad; Automatización; Infrastructure&#8209;as&#8209;code; Managed databases; Modern frameworks; Modular architecture
       image-path: /assets/custom/img/service-lines/software-modernisation/tabber-technology.svg
 
     - id: aproximacion-de-negocio
-      title: Aproximación de negocio
+      title: Aproximación de Negocio
```

Comment:
I would check with Helena on capitalising more than the first word in Spanish. I think they have different capitalisation rules.

Reply:
I hear you. There's this thread in Slack. https://codurance.slack.com/archives/C0136BDL3FW/p1593010949028000
We can always change it back if needed but for now @nicholemellekas has asked for it and we've found some rules to suggest it will be ok

Reply:
👍